packages/squiggle-lang/package.json

```diff
@@ -7,11 +7,10 @@
     "bundle": "webpack",
     "start": "rescript build -w -with-deps",
     "clean": "rescript clean",
-    "test:reducer": "jest --testPathPattern '.*__tests__/Reducer.*'",
+    "test:reducer": "jest __tests__/Reducer*/",
```

Comment:
Was this intended?

Reply:
Heck, a lot of this was unintended. Sorry

Reply:
Oh, yeah it seems to be done by yarn. For some reason we specified `test:reducer` twice, and it's chosen the second one to write in the file. It's also alphabetically sorted the packages. So nothing has been changed.